### DISCUSSION TITLE
Pikaviestiohjelma Element/Matrix ei toimi

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#1165] [Tiedotteet]
Koha-Suomen käyttämä Matrix-verkossa toimiva Element-pikaviestiohjelma on ollut puolesta päivästä saakka alhaalla, eikä viestejä pysty lähettämään eikä vastaanottamaan. Katkon syy ja kesto ei ole tiedossa. [Matrix status](https://status.matrix.org/) -sivu.

Koha-Suomi käyttää Element-pikaviestiohjelmaa oman henkilökunnan keskinäiseen viestintään sekä viestintään pääkäyttäjien kanssa.

**Comment by emta001** [comment]
Matrixin palvelin on taas toiminnassa.
